deploy/chart/devfile-registry/templates/configmap.yaml

```diff
@@ -42,10 +42,3 @@ data:
         prometheus:
           enabled: true
           path: /metrics
-
-  devfile-registry-hosts.json: |
```

Comment:
Do we not need to include this anymore when deploying the viewer? What's the host name used for page regeneration if this is left out?

CC @schultzp2020

Reply:
This is replaced by env vars `SITE_URL` and `NEXT_PUBLIC_DEVFILE_REGISTRIES` defined in the registry viewer container.